copy-reports/copy-reports.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+set -ex
+set -euo pipefail
```

Comment:
you can always do euxo to just get this on one line. if you are on the fence about x, dont use it, but I think scripts people run at home, it is always nice to show them what you are doing

Reply:
^^^^ absolutely
`set -euxo pipefail`